android/src/main/java/io/invertase/googleads/ReactNativeAppModule.java

```diff
@@ -0,0 +1,173 @@
+package io.invertase.googleads;
```

Comment:
Oh wow! Did not realize I completely missed part of the module loading infrastructure. Oof